src/js/ripple/amount.js

```diff
@@ -1,12 +1,16 @@
+ 'use strict';
+
+ /*eslint-disable new-cap*/
```

Comment:
Instead of disabling, can we rename wherever this `new-cap` setting was throwing an error? We reached consensus on the rules, disabling them instead of following throws away that effort. If we want to our code to look like what we agreed on, we need to make some changes. It doesn't seem like it's a big change unless I'm missing something.